qemu/tests/x86_cpu_model.py

```diff
@@ -75,6 +75,20 @@ def run(test, params, env):
         test.fail("Guest cpu model is not right")
 
     if params["os_type"] == "linux":
+        check_cmd = params.get("check_cmd")
+        if check_cmd:
+            vulnerabilities = process.getoutput(
+                params.get("vulnerabilities").strip()).split()
```

Comment:
Had better get guest and host vulnerabilities respectively and only compare the common ones.

Reply:
If I were you, I would create a dict as below, and record guest and host vulnerabilities in log.
{"itlb_multihit": "Not affected", "l1tf": "Not affected", ...}

Reply:
> Had better get guest and host vulnerabilities respectively and only compare the common ones.
Is it possible that guest have more vulnerabilities than host?

Reply:
@huangyum update, please help to review.